package.json

```diff
@@ -6,6 +6,7 @@
   "style": "themes/prism.css",
   "scripts": {
     "build": "gulp",
+    "generate-docs": "gulp docs",
```

Comment:
Maybe?

```suggestion
    "docs:generate": "gulp docs",
```

Reply:
Actually!
```suggestion
    "build:docs": "gulp docs",
```

Reply:
I went with `generate-docs` because that's [what our theme suggested](https://github.com/clenemt/docdash#usage-npm).

Reply:
Actually, `build:docs` was my first thought as well. I decided against it because people might think that they also need to run the `build:docs` task if they were told to "rebuild Prism".

Reply:
I don't think that confusion is super likely & isn't terrible if it does happen, and `build:docs` is more consistent with how we define other tasks.

Reply:
Should it be part of the default build task then? (To be consistent with `test` and `test:*`.)

Reply:
Doesn't have to be, imo.

Reply:
It might be better to have it be part of the default build task. I really want to avoid the case whether somebody makes some changes to `prism-core.js` (or any of the file from which the doc is generated from) and then the next person wants to make a PR, runs the `build:docs` task and then has to ask whether they should also commit that as well.

Let's make part of the default build task. If it's too slow or causes problems otherwise, we can kick it out later.